data/deep missions.txt

```diff
@@ -339,25 +339,24 @@ mission "Deep: Mystery Cubes 0"
 			
 			branch interested
 				has "deep: interested in cubes"
-			label interested
+			
 			choice
 				`	"Is there anything else that you need me to do?"`
 					goto next
-				`	"What is it that those cubes are for?"`
```

Comment:
I know that using `label interested` twice was an error, but I'm talking about removing the ability to ask what the cubes are the second time around if you didn't ask the first time.